assembly/ScaleString.ts

```diff
@@ -30,6 +30,27 @@ export class ScaleString extends ByteArray {
             this.values[i] = u8Input[i];
         }
     }
+    /**
+     * @description Non-static constructor method used to populate defined properties of the model
+     * @param bytes SCALE encoded bytes
+     * @param index index to start decoding the bytes from
+     */
+    populateFromBytes(bytes: u8[], index: i32 = 0): void{
```

Comment:
Can we extract this logic into `_` private functions for internal use in order to remove the code duplication?